src/toil_scripts/tools/preprocessing.py

```diff
@@ -131,9 +228,63 @@ def run_gatk_preprocessing(job, cores, bam, bai, ref, ref_dict, fai, phase, mill
     return pr.rv(0), pr.rv(1)
 
 
-def run_realigner_target_creator(job, cores, bam, bai, ref, ref_dict, fai, phase, mills, mem, unsafe=False):
+def run_germline_preprocessing(job, cores, bam, bai, ref, ref_dict, fai, phase, mills, dbsnp,
```

Comment:
Document `mock`. Split params out to one per line unless there is a logical grouping.